# GetPath.srv

# --- request constants (Field type + Path name)
uint8 RIGHT_START=1
uint8 RIGHT_DAIZA_COLLECT=2
uint8 RIGHT_DAIZA_PLACE=3
uint8 RIGHT_HINA_ZONE=4
uint8 RIGHT_HINA_COLLECT=5
uint8 RIGHT_HINA_PLACE=6
uint8 RIGHT_HINA_PLACE_RETRY=7
uint8 RIGHT_DAIZA_PLACE_RETRY=17
uint8 RIGHT_HINA_COLLECT_RETRY=18
uint8 RIGHT_BONBORI_BACK=20

uint8 LEFT_START=8
uint8 LEFT_DAIZA_COLLECT=9
uint8 LEFT_DAIZA_PLACE=10
uint8 LEFT_HINA_ZONE=11
uint8 LEFT_HINA_COLLECT=12
uint8 LEFT_HINA_PLACE=13
uint8 LEFT_HINA_PLACE_RETRY=14
uint8 LEFT_DAIZA_PLACE_RETRY=15
uint8 LEFT_HINA_COLLECT_RETRY=16
uint8 LEFT_BONBORI_BACK=19

# ---

uint8 path_number

---
Path2DWithParams path
int32[] indices
int8[] daiza_commands
int8[] hina_commands
bool[] bonbori_commands